src/App/App.scss

```diff
@@ -0,0 +1,135 @@
+@import '../scss/variables';
+
+body {
+  color: $font-color;
+  background-color: $background-color;
+}
+
+a {
+  color: $font-color;
+  text-decoration: none;
+}
+
+a:hover {
+  color: $hover-color;
+}
+
+.active:hover {
+  color: $font-color;
+  text-decoration: underline;
+  cursor: default;
+}
+
+.active {
+  text-decoration: underline;
+}
+
+.btn-primary {
+  padding-bottom: 0;
+  margin-bottom: -0.1em;
+  padding-left: 0;
+  background-color: $background-color;
+  border-color: $background-color;
+}
+
+.btn-primary:hover {
+  cursor: pointer;
+}
+
+.show>.btn-primary.dropdown-toggle {
+  padding-bottom: 0;
+  margin-bottom: -0.1em;
+  padding-left: 0;
+  background-color: $background-color;
+  border-color: $background-color;
+}
+
+.genre-filter {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0.9em 0 0.75em 0;
+}
+
+// .active-text {
+//   text-decoration: underline;
+// }
+
+.btn {
+  padding: none;
+}
+
+button {
+  border: none;
+}
+
+.clear {
+  height: 2em;
+  border: solid $font-color 0.1em;
+  border-radius: 1em;
+  color: $font-color;
+  background-color: $background-color;
+  font-size: 0.8em;
+  outline: none;
+}
+
+.dropdown-menu.show {
+  background-color: $background-color;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  height: 18em;
+  width: 14em;
+}
+
```

Comment:
You're going to need to make this css more specific to target only the dropdown you want. Add a class to the dropdown component that you want to customize. I'm not sure what it will look like in the end but I imagine the selector will look something like this:
```
.my-custom-dropdown > .dropdown-menu.show {
  height: 18em
}
```
Bootstrap will generate some wrapping divs and stuff, so you just need to drill down into them. This part of the inspector can sometimes be helpful in finding the right selector:

![Screen Shot 2021-01-26 at 9 10 32 AM](https://user-images.githubusercontent.com/16562801/105871334-6e0d8b00-5fb6-11eb-98da-333b73c4b829.png)